packages/phone-number-privacy/common/src/interfaces/responses.ts

```diff
@@ -13,13 +13,27 @@ import {
 import { Domain, DomainState } from '../domains'
 
 // Phone Number Privacy
-export interface SignMessageResponseSuccess {
+export interface PnpState {
```

Comment:
Optional Nit: Consider PnpQuotaStatus for consistency

Reply:
initially named it this way to be consistent with `DomainState` but do like `PnpQuotaStatus` as well

Reply:
oh wait there is a `PnpQuotaStatus` that already exists in signer 😱